rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -375,13 +375,55 @@ class InternalInterpreter(val systemInterface: SystemInterface) {
                 is ZAddStmt -> {
                     assign(statement.target, eval(statement.expression))
                 }
+                is AddStmt -> {
```

Comment:
You could move this logic to a separate method, as we discuss during the last design session

Reply:
Ok, we're working on it.
Thanks